### PR TITLE
Errno exception handling

### PR DESCRIPTION
**Pull request by jaykumar-jnpr**
#### Proposed solution to following

[sensu-plugins-filesystem-checks/issues/14](https://github.com/sensu-plugins/sensu-plugins-filesystem-checks/issues/14)
#### Purpose

Adds few Errno exception handling to **check-fs-writable** check
- Refer [errno-exceptions](http://blog.honeybadger.io/understanding-rubys-strange-errno-exceptions/) for more details on Errno types
- Errno types included in this pull request are only specific to what errors the check might encounter. Other/out-of-ordinary exceptions remains untouched.
#### Known Compatablity Issues

None I am aware of.

### CODE REVIEW

**Comment by sstarcher** [comment]
can you fix the travis-ci failures

File: bin/check-fs-writable.rb [hunk 1]
```diff
@@ -143,6 +143,9 @@ def manual_test
       file.close
       file.unlink
     end
+  rescue Errno::EPERM, Errno::ENOENT, Errno::EIO, Errno::EACCES, Errno::ENOSPC, Errno::EROFS => e
+  $stderr.puts "Caught the exception: #{e}"
+  exit -1
```
Comment (majormoses) [inline bin/check-fs-writable.rb:148]: I dont think we should exit with `-1` if anything it should be a 1 but anyways I also feel like this should be either an `unknown`, `warning`, or `critical` which also would remove needing to specifically redirect the output to stderr.

File: bin/check-fs-writable.rb [hunk 2]
```diff
@@ -143,6 +143,9 @@ def manual_test
       file.close
       file.unlink
     end
+  rescue Errno::EPERM, Errno::ENOENT, Errno::EIO, Errno::EACCES, Errno::ENOSPC, Errno::EROFS => e
+  $stderr.puts "Caught the exception: #{e}"
```
Comment (majormoses) [inline bin/check-fs-writable.rb:147]: see comment below, anyways whatever goes here should be indented properly.

**Review by majormoses — CHANGES_REQUESTED** [review]

**Comment by majormoses** [comment]
@jaykumar-jnpr are you planning on seeing this through to completion?

**Comment by majormoses** [comment]
closing due to inactivity

**Pull request closed without merging**